### DISCUSSION TITLE
USB flashing for demo units

### DISCUSSION THREAD

**Discussion by Aleksiej-13** [pembem22/connect-plus#70] [Ideas]
Is there any possibility to change firmware on jbl demo unit? I have a flip 5 demo it works only when plugged to usb and works as usb speaker i disassembled it it have a battery microphone bt module with antenna it looks just like retail version but it cant go to normal mode i pressed all buttons combinations i think its just firmware

**Comment by shortrockerz** [comment]
Bluetooth + Play button 10s

**Reply by Aleksiej-13** [reply]
nope i tried not working i bought it as new it has a sticker on the box "demo unit" and cant get out from usb mode

**Reply by shortrockerz** [reply]
You need to do that when the speaker is powered on usb

**Reply by Aleksiej-13** [reply]
im doing it right now on usb power still nothing happend after 40s

**Reply by AZSkeleton** [reply]
same problem on my flip 5, any news ?
